flink-tests/src/test/java/org/apache/flink/test/runtime/ShuffleCompressionITCase.java

```diff
@@ -84,8 +84,6 @@
     @Test
     public void testDataCompressionForBoundedBlockingShuffle() throws Exception {
```

Comment:
After changing the default value to true, I guess many test cases including BlockingShuffleITCase have already covered the scenario of data compression. Here what we really need is a test case which covers the scenario where data is not compressed. Could you please change the method name including the below one to something like testBoundedBlockingShuffleWithoutCompression/testSortMergeBlockingShuffleWithoutCompression, and set the bool flag of data compression to false?